subroom.lua

```diff
@@ -13,7 +13,7 @@ function OnStartup()
 		sPath = Core.GetPtokaXPath().."scripts/",
 		sDepPath = "dependency/",
 		sPickleFile = "pickle.lua",
-		sLogPath = "/www/ChatLogs/"..os.date( "%Y/" ),
```

Comment:
The removed `%Y` placeholder from here should be incorporated back while saving the file.